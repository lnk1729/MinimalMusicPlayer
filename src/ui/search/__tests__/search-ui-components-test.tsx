import React from 'react';
import renderer from 'react-test-renderer';
import SearchLander from '../searchlander.component';
import SearchNoResults from '../searchnoresults.component';

test('SearchLander renders correctly', () => {
  const tree = renderer.create(<SearchLander />).toJSON();
  expect(tree).toMatchSnapshot();
});

test('Searchnoresults renders correctly', () => {
  const tree = renderer.create(<SearchNoResults />).toJSON();
  expect(tree).toMatchSnapshot();
});
