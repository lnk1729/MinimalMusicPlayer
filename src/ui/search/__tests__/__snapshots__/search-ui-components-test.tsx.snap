// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SearchLander renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "flexDirection": "column",
      "marginTop": "50%",
      "width": "100%",
    }
  }
>
  <Text
    style={
      Object {
        "color": "#FFFFFF",
        "fontSize": 20,
        "fontWeight": "700",
        "textAlign": "center",
      }
    }
  >
    Play some music!
  </Text>
  <Text
    style={
      Object {
        "color": "hsla(0, 0%, 80%, 1)",
        "fontSize": 16,
        "marginTop": 4,
        "textAlign": "center",
      }
    }
  >
    Search for your favorite songs
  </Text>
</View>
`;

exports[`Searchnoresults renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "flexDirection": "column",
      "marginTop": "50%",
      "width": "100%",
    }
  }
>
  <Text
    style={
      Object {
        "color": "#FFFFFF",
        "fontSize": 20,
        "fontWeight": "700",
        "textAlign": "center",
      }
    }
  >
    Couldn't find a match
  </Text>
  <Text
    style={
      Object {
        "color": "hsla(0, 0%, 80%, 1)",
        "fontSize": 16,
        "marginTop": 4,
        "textAlign": "center",
      }
    }
  >
    Try searching again with a different spelling or keyword
  </Text>
</View>
`;
